docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -149,6 +153,8 @@ Some possibly useful slots:
 
 - `sidebarFooter`: allows you to add footer content in the sidebar.
 
+- `branding`: allows you to pass in your own component in the branding section.
```

Comment:
Maybe call it "app title" instead of "branding" in my first suggestion to match the description here more https://github.com/mui/toolpad/pull/4477/files#diff-87af69cfab4c32bae2f8f22cd3d491883936a394263a91aea7824875db51e2a7